From 783689b201689dd49605faa9f20622e19c273072 Mon Sep 17 00:00:00 2001
From: Jason Dere <jdere@hortonworks.com>
Date: Wed, 21 Oct 2015 10:56:52 -0700
Subject: [PATCH 1007/1164] CDH-51248 HIVE-12179: Add option to not add
 spark-assembly.jar to Hive classpath (Jason Dere,
 reviewed by Ashutosh Chauhan)

(cherry picked from commit 90aa6b49afc884913b36a8269f6c02431672b144)

Change-Id: If34c9626c97ab833b78bd406c7d55959147735ed
---
 bin/hive |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/hive b/bin/hive
index b39949e..6d1b562 100755
--- a/bin/hive
+++ b/bin/hive
@@ -109,7 +109,7 @@ for f in ${HIVE_LIB}/*.jar; do
 done
 
 # add Spark assembly jar to the classpath
-if [[ -n "$SPARK_HOME" ]]
+if [[ -n "$SPARK_HOME" && !("$HIVE_SKIP_SPARK_ASSEMBLY" = "true") ]]
 then
   sparkAssemblyPath=`ls ${SPARK_HOME}/lib/spark-assembly-*.jar`
   CLASSPATH="${CLASSPATH}:${sparkAssemblyPath}"
-- 
1.7.9.5

